### DISCUSSION TITLE
Is there a way to preload extensions every time I fire up Duckdb CLI?

### DISCUSSION THREAD

**Discussion by ddanieltan** [duckdb/duckdb#4883] [Q&A] [answered]
Slightly irritated that I have to always run `LOAD parquet;` or `LOAD json;` every time I fire up the cli. Any way for me to pre-configure these to always load?

**Comment by Mytherin** [comment, marked as answer]
The `CLI` reads from ` ~/.duckdbrc` on startup. I personally have it configured to change the prompt and nullvalue, but you can put any commands you want in there:

```bash
.prompt 'D '
.null NULL
```

**Reply by ddanieltan** [reply]
Perfect! thank you
